strawberry/resolvers.py

```diff
@@ -78,7 +80,8 @@ def get_result_for_field(
 
 
 def get_resolver(field: FieldDefinition) -> Callable:
-    def _check_permissions(source, info, **kwargs):
+    # TODO: convert info to our info
+    def _check_permissions(source, info: Info, **kwargs):
```

Comment:
What does this TODO mean?

Reply:
the info that it is passed here comes from graphql-core, I was thinking we might need to convert it when and if our version starts differing or even now (it might be a bit weird to inspect the info parameter and see that it is a different type)

Reply:
Let's open an issue and link it here :+1:

Reply:
done!